db/schema.rb

```diff
@@ -624,6 +624,7 @@
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
     t.boolean "platform_authenticator"
+    t.string "transports", array: true
```

Comment:
In theory, `transports` could possibly make `platform_authenticator` field redundant, since you could maybe derive that, e.g.

```rb
def platform_authenticator?
  transports != ['usb']
end
```

Reply:
maybe `nfc` too in there?

Reply:
Yeah, it probably gets more complex in real-world usage, hence I'd advocate for keeping the boolean for now.